contrib/completion/bash/docker-compose

```diff
@@ -131,25 +131,25 @@ _docker-compose_kill() {
 			COMPREPLY=( $( compgen -W "--help -s" -- "$cur" ) )
 			;;
 		*)
-			__docker-compose_services_running
+			__docker_compose_services_running
 			;;
 	esac
 }
 
 
-_docker-compose_logs() {
+_docker_compose_logs() {
 	case "$cur" in
 		-*)
 			COMPREPLY=( $( compgen -W "--help --no-color" -- "$cur" ) )
 			;;
 		*)
-			__docker-compose_services_all
+			__docker_compose_services_all
 			;;
 	esac
 }
 
 
-_docker-compose_migrate-to-labels() {
+_docker_compose_migrate_to_labels() {
```

Comment:
As `migrate-to-labels` contains dashes, completion for it does not work. The generic approach for identifying subcommands expects `_docker_compose_migrate-to-labels()`. An additional transformation from `-` to `_` is needed in the subcommand parsing routine.

Please change
```
local completions_func=_docker_compose_${command}
```
to
```
local completions_func=_docker_compose_${command//-/_}
```